### DISCUSSION TITLE
Which version of urllib3 is used in https://pypi.org/project/botocore/1.27.9/ ?

### DISCUSSION THREAD

**Discussion by AnirKulk** [boto/botocore#2697] [Q&A] [answered]
Which version of urllib3 is used in https://pypi.org/project/botocore/1.27.9/ ?

**Comment by nateprewitt** [comment, marked as answer]
Hi @AnirKulk, Botocore supports [this range](https://github.com/boto/botocore/blob/daf4110dc9be6f87fdaef0992499896c95997f91/setup.py#L28) of versions for urllib3. If a version in that range is already installed, botocore will generally use that version. Otherwise, pip will download the latest available version for your version of Python (likely urllib3 1.26.9).
